File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/thread_message_list_response.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class ThreadMessageListResponse:
    """OpenAI thread message list response"""
    object: str = None
    data: List[ThreadMessageResponse] = None
    has_more: bool = None
    first_id: str = None
    last_id: str = None

File: sdkwork-ai-sdk-go/types/thread_message_list_response.go
package types

// OpenAI thread message list response
type ThreadMessageListResponse struct {
	Object string `json:"object"`
	Data []ThreadMessageResponse `json:"data"`
	HasMore bool `json:"has_more"`
	FirstId string `json:"first_id"`
	LastId string `json:"last_id"`
}
